metagraph/src/graph/alignment/aligner_extender_methods.cpp

```diff
@@ -237,10 +237,20 @@ inline void update_del_scores(const DBGAlignerConfig &config,
 
 #ifdef __AVX2__
 
+// Drop-in replacement for _mm_loadu_si64
+inline __m128i mm_loadu_si64(const void *mem_addr) {
+    return _mm_loadl_epi64((const __m128i_u*)mem_addr);
+}
+
+// Drop-in replacement for _mm_storeu_si64
+inline void mm_storeu_si64(void *mem_addr, __m128i a) {
+    _mm_storel_epi64((__m128i_u*)mem_addr, a);
+}
```

Comment:
why not use `_mm_loadu_si64` when it's defined?

Reply:
I think it's actually better to do it this way. According to the Intel Intrinsics guide, `_mm_loadu_si64` has much higher latency than `_mm_loadl_epi64`. The code I have above is actually how gcc implements `_mm_loadu_si64`.

Plus, implementing a check for `_mm_loadu_si64`s existence would be more annoying since we'd have to do it with cmake.

Reply:
I don't think it's worth further complicating the CMakeLists.txt for just this source file.